llpc/lower/llpcSpirvProcessGpuRtLibrary.cpp

```diff
@@ -641,16 +642,93 @@ void SpirvProcessGpuRtLibrary::createGetStaticId(llvm::Function *func) {
   m_builder->CreateRet(m_builder->create<GpurtGetRayStaticIdOp>());
 }
 
+// =====================================================================================================================
+// Create function to get known set ray flags
+//
+// @param func : The function to create
```

Comment:
This is not really creating a function

Reply:
Yeah, we are actually filling functions, but I'm just following the comments everywhere else in this file.

Reply:
I suggest changing the other comments then as well, since it might be confusing.